caas/kubernetes/workload/detectors-k8s-workload.tf

```
@@ -0,0 +1,82 @@
+resource "signalfx_detector" "heartbeat" {
+	name = "${join("", formatlist("[%s]", var.prefixes))}[${var.environment}] Kubernetes API server heartbeat"
```

Same name as https://github.com/claranet/terraform-signalfx-detectors/pull/23/files#diff-7d36032359e68de3850038e41edeae9cR2
```suggestion
	name = "${join("", formatlist("[%s]", var.prefixes))}[${var.environment}] Kubernetes workload heartbeat"
```

---

updated